src/components/terminal.rs

```diff
@@ -1,21 +1,24 @@
-use crate::types::terminal_content::TerminalContent;
+use crate::types::level::Level;
 use leptos::{ev::SubmitEvent, html::Input, *};
 use std::time::Duration;
 
+// TODO - Tech Debt: Find a way to not allow the whole
+// component but rather the single closure
+#[allow(clippy::redundant_closure)]
```

Comment:
So many items in Leptos rely on closures to update values, this one might just need to go on all components.